.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We definitely don't need git to track this file.

Reply:
I'm not entirely sure about your first comment. Can you teach me more about it when we meet? thanks a lot for fixing it btw! ^_^